AGENTS.md

```
@@ -0,0 +1,3 @@
+- never run publint with pnpm, instead always use `npx publint --pack npm`
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)